binding/unity/Assets/Rhino/RhinoManager.cs

```diff
@@ -39,7 +39,7 @@ public class RhinoManager
         /// <param name="requireEndpoint">
         /// Boolean variable to indicate if Rhino should wait for a chunk of silence before finishing inference.
         /// </param>
-        /// <param name="errorCallback">(Optional) Callback that triggers is the engine experiences a problem while processing audio.</param>
+        /// <param name="processErrorCallback">(Optional) Callback that triggers is the engine experiences a problem while processing audio.</param>
```

Comment:
(the grammar on the description I wrote is awful...) "(Optional) Reports errors that are encountered while the engine is processing audio"